docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -0,0 +1,131 @@
+[id="deploying-a-ceph-ingress-daemon_{context}"]
+
+= Deploying a {Ceph} ingress daemon
+
+To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_{context}[{Ceph} daemon cardinality].
+
+`HAProxy` is managed by {OpenStackPreviousInstaller} through `Pacemaker`: the three running instances at this point will point to the old RGW backends, resulting in a broken configuration. Since you are going to deploy the https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/ansible_plugins/modules/ceph_mkspec.py[Ceph Ingress Daemon], the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
+//kgilliga: I will eventually rewrite this intro. I don't think we should link to GitHub downstream.
```

Comment:
remove the github link downstream, while upstream should point to wallaby because master has been recently retired